webcomponents/tool/src/main/frontend/js/tasks/sakai-tasks-create-task.js

```diff
@@ -224,6 +224,12 @@ export class SakaiTasksCreateTask extends SakaiDialogContent {
     this.assignationType = "user";
     this.mode = "create";
     this.siteId = this.siteIdBackup;
+
+    // Clear the value of the datetime picker input
+    const datePickerInput = datePicker.querySelector('input[type="datetime-local"]');
+    if (datePickerInput) {
+      datePickerInput.value = '';
+    }
```

Comment:
I think you don't have to query the element it's already there, you could just add this line: `datePicker.value = "";`

Reply:
Hi Kunal, ```datePicker.value = ''``` does not work in this case. I think we need to modify the <input> tag with id='date-picker-input-xxx'. Since the number in the Id is not defined in ```sakai-tasks-create-task.js```, I have to use ```datePicker.querySelector('input[type="datetime-local"]')``` to select this element.


<img width="1155" alt="Screenshot 2023-05-09 at 10 01 45 AM" src="https://github.com/sakaiproject/sakai/assets/102482288/74741478-b97b-4875-8065-7b2c57dd40d1">

Reply:
Maybe @adrianfish can help us.

Reply:
Zhuo's right - datePicker is the actual component, and the component has an input tag in its dom. However, I'd prefer him to add a reset function to sakai-date-picker so we can hide the implementation. So, pretend sakai-date-picker is shadow dom and don't select stuff from inside it.